lte/gateway/c/core/oai/tasks/ngap/ngap_state_converter.cpp

```diff
@@ -24,26 +24,26 @@ using namespace std;
 using namespace magma::lte;
 using namespace magma::lte::oai;
 
-using magma::lte::oai::Ngap_UeDescription;
-using magma::lte::oai::NgapState;
```

Comment:
What is the motivation for this change? Mixing 4G and 5G terminology will make the code hard to read, e.g. storing gnb_ue_ngap_id as enb_ue_s1ap_id . Given that we are proposing to use the proto object directly in hashtables (https://github.com/magma/magma/issues/11199), this mixed naming will extend over the entire Ngap implementation. Let's keep the proto definitions separate for S1AP and NGAP.

Reply:
@ssanadhya, 
    we got feedback from @ulaskozat to reuse existing Proto, designed Stateless feature(#10805) such way to reuse existing Proto.
 cc: @panyogesh

Reply:
@ganeshg87 , could you please point me to the initial feedback comment?
Further, which changes in #10805 are you referring to in regards to reusing protos?

Reply:
@ssanadhya, 
    Sorry, PR Id was #10605 (not #10805) design document, amf stateless feature designed to reuse existing 4G infra.mme_nas_state.proto, nas_state.proto reused for amf.

Reply:
@ganeshg87 
We had an internal discussion with @ssanadhya and @ulaskozat, we agreed it's cleaner to maintain different protobuf structures for 4G and 5G. Main reasoning comes from the fact that we want to separate the naming of 5G and 4G representations once they are eventually extended and used as objects on the internal and redis datastore mappings. The same should be applied to PR #11302